images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -21,8 +22,10 @@ function Set-JavaPath {
     Write-Host "Set 'JAVA_HOME_${Version}_X64' environmental variable as $javaPath"
     setx JAVA_HOME_${Version}_X64 $javaPath /M
 
-    if ($Default)
+    if ($DefaultVersion)
```

Comment:
Why do we need to rename [switch]`$Default` parameter to [string]`$DefaultVersion`?
We don't use its value anywhere.